docs/index.rst

```diff
@@ -41,11 +41,13 @@ Project Info
 .. toctree::
    :maxdepth: 2
 
-   contributing
-   authors
+   CONTRIBUTING
```

Comment:
Let's keep the symlinks as lowercase so these references are all consistent?

Reply:
Impossible. In this case, links in MD docs would not work. 
I tested different combinations of uppercase\lowercase mapping. 
Current mapping is only one possible way to cover 99.5% links. (I know several links that would not work in on page mode, as RST and MD has different articles mapping)

So no changes in links.